action.yml

```
@@ -49,6 +60,26 @@ runs:
       run: pip install tutor==${{ inputs.tutor-version }}
       shell: bash
 
+    - if: ${{ inputs.extra-requirements == 'true' }}
+      name: Install extra requirements
+      run: |
+        # repositores is an array that contains private repositories.
+        repositories=("pearson-core")
+        for repository in "${repositories[@]}"; do
+          git clone https://${{ inputs.access-token }}@github.com/Pearson-Advance/$repository.git \
+            "$(tutor config printroot)/env/build/openedx/requirements/$repository"
+          echo "-e ./$repository" >> "$(tutor config printroot)/env/build/openedx/requirements/private.txt"
+          echo "Cloned "$repository" repository successfully."
+        done
+      shell: bash
+
+    - if: ${{ inputs.theme-repository != 'false' }}
```

Maybe you can change the default to `""`,  the conditional becomes:
```suggestion
    - if: ${{ inputs.theme-repository != '' }}
```
@sergivalero20